contributing.md

```diff
@@ -16,6 +14,22 @@ dotnet cake --target=Test
 
 The project is built with `dotnet-boxed`, so if you have any questions to the structure, check out [this article](https://rehansaeed.com/the-fastest-nuget-package-ever-published-probably/)
 
+# Building the documentation locally
+To build the documentation locally to test out things you will need to install doxygen.
+Installation instructions can be found [here](https://www.doxygen.nl/download.html) (scroll down to "Sources and Binaries")
+
+After having installed doxygen you can generate the documentation by running:
```

Comment:
It worked fine, I kind of figured that it needs to be run from the location of the doxygen-config.toml, but just in case maybe it is worth adding that specification that it needs to be run from root? 

p.s. the suggestion is not best formulated.  

```suggestion
After having installed doxygen you can generate the documentation by running the bellow command from the root folder of the project:
```